modules/gdscript/gdscript_parser.h

```diff
@@ -50,6 +50,31 @@
 #include "gdscript_warning.h"
 #endif // DEBUG_ENABLED
 
+static Variant::Type _variant_type_to_typed_array_element_type(Variant::Type p_type) {
```

Comment:
I'm not really fond of adding this function to the header. This should be added to the `.cpp` instead and the methods that depend on it can be defined in the `.cpp` too since they're not being inlined anyway.

Reply:
Fixed.